ts/features/itwallet/common/components/discoveryBanner/ItwDiscoveryBanner.tsx

```diff
@@ -34,10 +39,17 @@ export const ItwDiscoveryBanner = ({
 }: ItwDiscoveryBannerProps) => {
   const bannerRef = createRef<View>();
   const dispatch = useIODispatch();
-
+  const isWalletRemotelyActive = useIOSelector(
+    itwIsWalletInstanceRemotelyActiveSelector
+  );
   const navigation = useIONavigation();
   const route = useRoute();
 
+  const [bannerState, setBannerState] = useState<{
+    shouldRender: boolean;
+    type: BannerType | null;
+  }>({ shouldRender: false, type: null });
```

Comment:
Do we really need `shouldRender`? When the banner type is null we don't render anything, as soon as it is either `reactivating` or `onboarding` we render the appropriate type.